client/src/containers/EditProfile/EditUserProfile.js

```diff
@@ -35,7 +35,6 @@ class EditUserProfile extends React.Component {
     formData.append('username', formValues.username.value);
     formData.append('first_name', formValues.firstname.value);
     formData.append('last_name', formValues.lastname.value);
-    formData.append('email', formValues.email.value);
```

Comment:
Why are we extracting the data and appending it to formData in component? 
the practice being followed in rest of the app is to collect the data in component and then pass it to the service, 
the service will be responsible for transformation of data in which ever form it is required.